spec/system/show_work_spec.rb

```diff
@@ -145,9 +145,11 @@
 
     # Title table
     within('table#title-table') do
-      expect(page).to have_css('caption', text: 'Title')
+      expect(page).to have_css('caption', text: 'Title and contact')
```

Comment:
Is the discrepancy between "Title and contact" and "Title & contact" deliberate?

Reply:
I assume you mean between the tab label and the show page? Yes, they are different in the figma designs.